enhancements/machine-api/machine-instance-lifecycle.md

```diff
@@ -135,6 +135,10 @@ For it to happen in a non disruptive manner for actuators this proposal is inten
 
 Once this is settled we might consider to simplify the interface.
 
+### Machine lifecycle diagram
+
```

Comment:
can we please make this a sub-section of the "Implementation Details" After "Errors"?